### ISSUE TITLE
1822 missing token

### ISSUE THREAD

**Issue by russrusnak** [rust-lang/rust#103371]
In 1822 game id 98155, the NER's destination token disappeared from the board even though it shows as there on the entities page.

I know nothing about coding, I am just trying to report a possible bug in an 1822 game on 18xx.games

Thank you for filing a bug report! 🐛 Please provide a short summary of the bug,
along with any information you feel relevant to replicating the bug.
-->

I tried this code:

```rust
<code>
```

I expected to see this happen: *explanation*

Instead, this happened: *explanation*

### Meta

`rustc --version --verbose`:
```
<version>
```

<details><summary>Backtrace</summary>
<p>

```
<backtrace>
```

</p>
</details>

**Comment by compiler-errors** [comment]
@russrusnak -- this is the [Rust programming language](https://www.rust-lang.org/). I don't think this issue is reported to the right place. Hope you find a fix for your issue!
